core/src/main/java/org/sonarsource/sonarlint/core/telemetry/payload/TelemetryPayload.java

```diff
@@ -79,12 +84,14 @@
   private final TaintVulnerabilitiesPayload taintVulnerabilitiesPayload;
 
   @SerializedName("rules")
-  private TelemetryRulesPayload telemetryRulesPayload;
+  private final TelemetryRulesPayload telemetryRulesPayload;
+
+  private transient final Map<String, Object> additionalAttributes;
```

Comment:
Nitpick (and flagged on Next): modifiers are not in JLS-recommended order

Reply:
Fixed